modules/auxiliary/scanner/http/backup_file.rb

```diff
@@ -23,7 +23,7 @@ def initialize(info = {})
 
     register_options(
       [
-        OptString.new('PATH', [ true,  "The path/file to identify backups", '/index.asp'])
+        OptString.new('PATH', [ true,  "The path/file to identify backups, use ':' as a delimiter for multi-path/file input", '/index.asp'])
```

Comment:
I don't have the full context on this module; Does this module need to work with full paths for windows envs? If so I don't think this will work for this scenario: `C:\\inetpub\\wwwroot\\foo.asp`

Potentially `PATH` is just a misnomer though, and this is really `FILE_LIST` or something

Reply:
Ah; gotcha. I believe the normal pattern is to take such wordlists from a file

So this would become `OptPath` and it would point towards a default wordlist that ships with metasploit, but the user can override it to point to their own wordlist

Like this:

https://github.com/rapid7/metasploit-framework/blob/e6fb1789f46b94c6eb4b2db595e5976355342ffd/modules/auxiliary/scanner/http/log4shell_scanner.rb#L59-L66 (I'd probably use this pattern for the file reading)

https://github.com/rapid7/metasploit-framework/blob/e6fb1789f46b94c6eb4b2db595e5976355342ffd/modules/auxiliary/scanner/http/ntlm_info_enumeration.rb#L29-L32 (Another example, but the file reading implementation isn't as good as the above)

Reply:
I see

We can migrate to the one `TARGET_URIS_FILE` as it more suitable - though none of those paths are relevant for backup files.. 

For best outcome we should use `dirb` (a tool used to bruteforce files and directories) common.txt file:
https://salsa.debian.org/pkg-security-team/dirb/-/blob/debian/master/wordlists/common.txt

Reply:
Hello @nrathaus, Thanks for your pr! Do you have any update on it?

Reply:
My last comment was a question, that never got answered, so I am not sure what is there update

Reply:
Moving to the file pattern sounds good to me; It looks like dirb's wordlist doesn't have `index.asp` - so that wouldn't have the same behavior as what the current module has

For MVP - what are your thoughts on creating a new wordlist with just index.asp in it to maintain backwards compatibility with the module's existing behavior, but doing so will now allow the user to provide their own list if the they want - i.e. common.txt etc

Reply:
@adfoster-r7 I don't think users need another file to maintain and update, if we are going to go on the 'wordlist' route, we should create a 'wordlist' that answers the requirements of multiple users of this file - there is no such thing as 'over testing' in my opinion, just because a wordlist for php also has asp references, isn't a bad thing